lib/stages/basebandReadout.cpp

```diff
@@ -586,7 +587,7 @@ void basebandReadout::write_dump(basebandDumpData data, basebandDumpStatus& dump
         .write(ftime0_offset);
     file.createAttribute<std::string>("type_time0_ctime", HighFive::DataSpace::From(time0_type))
         .write(time0_type);
-    double delta_t = (double)FPGA_PERIOD_NS / 1e9;
+    double delta_t = ts_to_double(Telescope::instance().seq_length());
```

Comment:
We've used `fpga_s` and `fpga_ns` in other files as part of this change. Maybe use the same?

Reply:
Makes sense.